src/reagent/impl/template.cljs

```diff
@@ -73,13 +73,23 @@
                            class))))
     p))
 
+(defn stringify-class [{:keys [class] :as props}]
+  (if (coll? class)
```

Comment:
I have used `coll?` in order to support sets actually (which is not covered by tests as I see it now, bummer). Do you think it's impractical/unnecessary?